galley/cmd/galley/cmd/server.go

```diff
@@ -151,6 +151,8 @@ func serverCmd() *cobra.Command {
 		serverArgs.SinkAuthMode, "Name of authentication plugin to use for connection to sink server.")
 	serverCmd.PersistentFlags().StringSliceVar(&serverArgs.SinkMeta, "sinkMeta",
 		serverArgs.SinkMeta, "Comma-separated list of key=values to attach as metadata to outgoing sink connections. Ex: 'key=value,key2=value2'")
+	serverCmd.PersistentFlags().BoolVar(&serverArgs.EnableServiceDiscovery, "enableServiceDiscovery", false,
```

Comment:
Do you also want to consume services, endpoints, pods, and nodes by default now? Or keep them filtered by default?

Reply:
They are already filtered by default (due to ExcludedTypes flag), if the flag is false.

Reply:
Right ... so you want to keep them filtered by default?

Reply:
Yes. The filtering is for serving side, not source.